hre/src/main/java/hre/config/Configuration.java

```diff
@@ -276,6 +276,7 @@ public static MessageProcessEnvironment getThisVerCors() throws IOException {
         env.addArg("-Xss128M");
         env.addArg("-cp", System.getProperty("java.class.path"));
         env.addArg("vct.main.Main");
+        env.setEnvironmentVar("TEMP", System.getenv("TEMP"));
```

Comment:
Can you guard this with `if(System.getenv("TEMP") != null)` please? I think the value is not defined in linux, so it will end up passing `TEMP=null` (the string "null").